staging/src/k8s.io/kube-aggregator/pkg/controllers/openapiv3/controllerv3.go

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
Did you need to call this file `controllerv3.go` since it's in the `openapiv3` package?

Reply:
ok, keeping as `controller.go`